text/0000-add-contributer-graph.md

```diff
@@ -0,0 +1,71 @@
+# Meta
+[meta]: #meta
+- Name: Add Contributer Graph
+- Start Date: 2020-03-08
+- Author(s): aemengo
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC proposes adding a customized contributer graph to the buildpacks [website](https://buildpacks.io/community/). Please see [mockup](#prior-art) for concept.
+
+# Definitions
+[definitions]: #definitions
+
+* Contribution - Opening a issue, Creating a PR, or Creating an RFC
+* Organization - The corporation represented by those making a contribution
+* OSS community - People engaging in discussion, usage and contribution to an OSS project
+
+# Motivation
+[motivation]: #motivation
+
+Tracking the health of an OSS community can be a very intensive activity. It can consist of keeping tabs on GitHub issues across multiple key repositories, or being an active participant in a multiple OSS slack channels, or keeping up with blog posts, etc. Although these collectively yield the most accurate insight into the health of an OSS community, not everyone can devote the time and attention on a consistent basis.
+
+For these others that are still invested in the health of the community, quantitative metrics are a useful recourse. These "others" could be prospective engineers deciding whether to immerse themselves within the community, or hobbyists looking to champion buildpacks within their organizations.
+
+The expected outcome is that more productive discussions about community engagement can occur when centered around the metrics attempting to depict it.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Please note: _This RFC does not intend to provide something fully realized. Instead, it aims to introduce the concept and have it be shaped by discussion, feasibility, and utility._
+
+At a high level, it is a graph that lives on the buildpacks [website](https://buildpacks.io/community/). The graph will show the full lifecycle of an OSS contribution, which could include the following:
+
+- RFC -> Issue/PR -> Shipped
```

Comment:
I think it might be a better workflow to have issue open > issue in progress (either as PR(s) or as an answer on the issue for support questions) > issue closed. A single issue can also span multiple or 0 PRs so we may essentially end up with a more complex graph here. 



We could have releases marking merged PRs that have been shipped.

Reply:
That was these issues can be tracked through different repositories by related links in github and you could annotate different components with different colors. That way the same lifecycle is applicable to an issue that starts as an RFC or as a support ticket.

Reply:
Very interesting. And you don't feel like information is lost?

You don't feel like we are omitting parts of the full journey for a given RFC?

Reply:
![IMG_20210314_222109__01.jpg](https://user-images.githubusercontent.com/16130816/111086373-b0157080-8513-11eb-8d3f-6dcc1efc4721.jpg)

I was imagining something like above to trace a lifetime of a single connected component in the contribution graph. You could tag each note with the person who open the issue/PR for example. 


People could drill down into the above from a higher level view like ![IMG_20210314_222053.jpg](https://user-images.githubusercontent.com/16130816/111086399-d804d400-8513-11eb-8ee8-0b0052f6d841.jpg)

This above summary/high level view could also lead to pie charts that can be pivoted in different ways like %age contributions by different organizations or users. 

LMK what you think :D

Reply:
You can also add more repositories on the left in their supposed order of dependency. For eg, we could add docs or samples to the bottom.

This way if a particular issue/PR has a lifecycle that starts and ends with docs, we can just hide other components.

Reply:
Sam, thanks so much. A picture is worth a thousand words.

These are great idea. Tbh, I don't know how I feel about pie charts w/ different pivots. Feels too "businessy". 

But I like what you're saying about higher level and then an in-depth view. I just want something appealing at that higher-level, that makes people want to click further. I'm thinking something more contributer-focused but not sure yet.